src/main/java/eu/enmeshed/model/request/requestItems/FreeTextRequestItem.java

```diff
@@ -11,7 +11,7 @@
 @Getter
 @Setter
 @SuperBuilder
-public class FreeTextRequestItem extends RequestItem {
-  Boolean accept;
-  String freeText;
+public class FreeTextRequestItem extends RequestItemDerivation {
+  private Boolean accept;
```

Comment:
Where does this `accept` property come from?

Reply:
it was there always.

Reply:
Should we remove it?

Reply:
it is :)